src/components/studies/participants/ParticipantSearch.tsx

```diff
@@ -72,7 +72,7 @@ const useStyles = makeStyles(theme => ({
 }))
```

Comment:
Did you make any changes in this file or is it just formatting?

Reply:
Just formatting